content/400-reference/200-api-reference/100-prisma-schema-reference.mdx

```diff
@@ -2010,20 +2017,21 @@ Defines a compound [unique constraint](../../../concepts/components/prisma-schem
 | Name     | Required | Type               | Description                                                                                               |
 | -------- | -------- | ------------------ | --------------------------------------------------------------------------------------------------------- |
 | `fields` | **Yes**  | `FieldReference[]` | A list of field names - for example, `["firstname", "lastname"]`. Fields must be mandatory - see remarks. |
-| `name`   | **No**   | `String`           | The name of the unique combination of fields - defaults to `fieldName1_fieldName2_fieldName3`             |
+| `name`   | **No**   | `String`           | The name of the unique combination of fields that is [reflected in the generated Prisma Client](#specify-a-custom-name-for-a-multi-field-unique-attribute) - defaults to `fieldName1_fieldName2_fieldName3`             |
+| `map` ([2.27.0](https://github.com/prisma/prisma/releases/tag/2.27.0) and later)  | No   | `String`             | The constraint name in the underlying database. Default naming convention: `{table_name}_{column_names}_uniq` (for example, `users_col1_col2_uniq`) |
 
 The name of the `fields` argument on the `@@unique` attribute can be omitted:
 
 ```prisma no-lines
 @@unique(fields: [title, author])
 @@unique([title, author])
-@@unique(fields: [title, author], name: "titleAuthor")
+@@unique(fields: [title, author], name: "titleAuthor", map: "myConstraintName")
```

Comment:
This makes it look a bit like the omitting was possible because the `map` was added - maybe a separate example for `map`?